### PR TITLE
default gc, title updates, some ux items

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#7685

*Description of changes:*
- some ux updates etc to misc registration types

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/views/newRegistration/ReviewConfirm.vue [hunk 1]
```diff
@@ -151,6 +152,9 @@ export default class ReviewConfirm extends Vue {
   }
 
   private get registrationTypeUI (): string {
+    if (this.getRegistrationType?.registrationTypeAPI === 'OT') {
```
Comment (kialj876) [inline ppr-ui/src/views/newRegistration/ReviewConfirm.vue]: enum pls

File: ppr-ui/src/views/newRegistration/LengthTrust.vue [hunk 2]
```diff
@@ -109,6 +110,9 @@ export default class LengthTrust extends Vue {
   }
 
   private get registrationTypeUI (): string {
+    if (this.getRegistrationType?.registrationTypeAPI === 'OT') {
```
Comment (kialj876) [inline ppr-ui/src/views/newRegistration/LengthTrust.vue]: enum pls

File: ppr-ui/src/views/newRegistration/AddParties.vue [hunk 3]
```diff
@@ -100,6 +101,9 @@ export default class AddParties extends Vue {
   }
 
   private get registrationTypeUI (): string {
+    if (this.getRegistrationType?.registrationTypeAPI === 'OT') {
```
Comment (kialj876) [inline ppr-ui/src/views/newRegistration/AddParties.vue]: enum pls

File: ppr-ui/src/views/newRegistration/AddCollateral.vue [hunk 4]
```diff
@@ -92,6 +93,9 @@ export default class AddCollateral extends Vue {
   }
 
   private get registrationTypeUI (): string {
+    if (this.getRegistrationType?.registrationTypeAPI === 'OT') {
```
Comment (kialj876) [inline ppr-ui/src/views/newRegistration/AddCollateral.vue]: enum pls

File: ppr-ui/src/components/collateral/Collateral.vue [hunk 5]
```diff
@@ -434,10 +435,15 @@ export default defineComponent({
 
     onMounted(() => {
       if (hasGeneralCollateral() && !collateral.generalCollateral) {
```
Comment (kialj876) [inline ppr-ui/src/components/collateral/Collateral.vue:437]: would be nice to have a test for whether or not general text is showing / the right general text is showing for this component for one without general collateral text, lien unpaid wages, and one with general collateral text for your changes below

**Review by kialj876 — CHANGES_REQUESTED** [review]
Hey Steph, looks good to me just a couple small things

**Review by doug-lovett — APPROVED** [review]
No further comments after Kial's review.

File: ppr-ui/src/components/collateral/Collateral.vue (hunk 5 above)
Comment (jordiwes) [reply ppr-ui/src/components/collateral/Collateral.vue:437]: updated enums and added tests

File: ppr-ui/src/views/newRegistration/ReviewConfirm.vue (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/src/views/newRegistration/ReviewConfirm.vue]: done

File: ppr-ui/src/views/newRegistration/LengthTrust.vue (hunk 2 above)
Comment (jordiwes) [reply ppr-ui/src/views/newRegistration/LengthTrust.vue]: done

File: ppr-ui/src/views/newRegistration/AddParties.vue (hunk 3 above)
Comment (jordiwes) [reply ppr-ui/src/views/newRegistration/AddParties.vue]: done

File: ppr-ui/src/views/newRegistration/AddCollateral.vue (hunk 4 above)
Comment (jordiwes) [reply ppr-ui/src/views/newRegistration/AddCollateral.vue]: done

File: ppr-ui/src/components/collateral/Collateral.vue (hunk 5 above)
Comment (kialj876) [reply ppr-ui/src/components/collateral/Collateral.vue:437]: awesome, thanks!

**Review by kialj876 — APPROVED** [review]
Looks good, thanks for making those changes. You just have 1 failing test to fix 😝 , but looks good once that is fixed

**Pull request merged**